controllers/hacker.controller.js

```diff
@@ -7,6 +7,32 @@ const Services = {
 };
 const Util = require("../middlewares/util.middleware");
 
+/**
+ * @async
+ * @function createHacker
+ * @param req
+ * @param res
+ * @return {JSON} Success or error status
+ * @description create a hacker from information in req.body.hackerDetails
+ */
+async function createHacker(req, res) {
+    const hackerDetails = req.body.hackerDetails;
+    
+    const success = await Services.Hacker.createHacker(hackerDetails);
+
+    if (success) {
+        return res.status(200).json({
+            message: "Hacker creation successful",
+            data: hackerDetails
+        });
+    } else {
+        return res.status(400).json({
```

Comment:
If there's an issue with creating a hacker, and data validation is totally fine, then this should be a 500 (internal error) error code. This should never happen, basically